docs/how-to/setting_up_a_survey.md

```diff
@@ -0,0 +1,70 @@
+This page provides an overview of how to utilize surveys in your OCS chatbot.
+
+### What are Surveys on Open Chat Studio?
+
+The Surveys feature allows chatbot makers to give users a link to a Google form (or any other link to a survey), both at the start and end of an OCS chatbot web session. 
+
+If you deploy your OCS chatbot on WhatsApp or Telegram, users will not be able to see the survey links or submit responses.
+
+- Incorporating survey questions in your prompt and structuring the prompt such that the chatbot starts and ends with questions as you would like it to. 
+- Sending users links to a Google form or other kind of survey directly, before or after providing them with the link to the chatbot. 
```

Comment:
These don't make sense in the current context. On confluence there is an additional sentence before this list:

```
If you deploy your OCS chatbot on WhatsApp or Telegram but would still like to capture pre- or post-survey questions, options include: 
```

Reply:
Ah yes, you are right. Must have gone missing with me committing suggestions from the bot.

Reply:
https://github.com/dimagi/open-chat-studio-docs/pull/28/commits/9d8120ae58ebeaafd677394401383d7932aa34bf